### ISSUE TITLE
Showing pet heals

### ISSUE THREAD

**Issue by Visitations** [AtheneGenesis/Vanilla_MikScrollingBattleText#3]
Any chance of adding the ability to show outgoing pet heals (totems) on party members?

**Comment by AtheneGenesis** [comment]
I don't have any Shaman to try this, I'll need some help to figure out if it's possible.
Reading only the full combatlog can you tell if the unit was healed by your totem and not one from another's raid/party member ?

**Comment by Visitations** [comment]
Yes, it shows they are being healed by my totem, however, doesn't show via MikScrolling.
